torch/csrc/jit/passes/python_print.cpp

```diff
@@ -1240,6 +1240,23 @@ struct PythonPrintImpl {
         body_ << name << " : " << type->python_str() << "\n";
       }
     }
+
+    size_t numConstants = moduleType->numConstants();
+    for (size_t i = 0; i < numConstants; i++) {
+      const auto& name = moduleType->getConstantName(i);
+      const auto& v = moduleType->getConstant(name).value();
+
+      indent();
+      body_ << name << " : " << "Final[" + v.type()->python_str() << "] = ";
+      if (v.isString()) {
```

Comment:
Similar here. This introduces a non-complete if-statement branch that will cause a huge number of reported issues in the future. This should use the already existing `printConstant` function.